apps/frontend/src/app/3_organisms/ZeroLocForm/utils.ts

```diff
@@ -10,49 +11,53 @@ import {
 } from './constants';
 import { AmountType } from './types';
 
+export const DEFAULT_RESERVE_FEE = Decimal.from(20);
```

Comment:
No magic numbers please - either use `LIQUIDATION_RESERVE_AMOUNT` from `constants.ts` or pull from `lib-base` instead, it's exposed as `ZUSD_LIQUIDATION_RESERVE`